.pre-commit-config.yaml

```
@@ -0,0 +1,14 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
+    hooks:
+    -   id: trailing-whitespace
+    -   id: end-of-file-fixer
+    -   id: check-yaml
+    -   id: check-added-large-files
+
+-   repo: https://github.com/psf/black
+    rev: 19.10b0
+    hooks:
+      - id: black
+        language_version: python3
```

I know `pre-commit` doesn't care about indentation, but this bothers me :rofl: 
>Sample [`pre-commit-config.yaml`](https://github.com/pre-commit/pre-commit/blob/master/.pre-commit-config.yaml).

```suggestion
repos:
-   repo: https://github.com/pre-commit/pre-commit-hooks
    rev: v3.2.0
    hooks:
    -   id: trailing-whitespace
    -   id: end-of-file-fixer
    -   id: check-yaml
    -   id: check-added-large-files

-   repo: https://github.com/psf/black
    rev: 19.10b0
    hooks:
    -   id: black
        language_version: python3
```

---

Done.